infra/Pos/Network/CLI.hs

```diff
@@ -194,67 +245,92 @@ monitorStaticConfig cfg@NetworkConfigOpts{..} origMetadata initPeers = do
           Right () -> return ()
 
     changedFormat, changedType, changedKademlia :: FilePath -> Text
-    changedFormat     = sformat $ "SIGHUP (" % shown % "): Cannot dynamically change topology."
-    changedType       = sformat $ "SIGHUP (" % shown % "): Cannot dynamically change own node type."
-    changedKademlia   = sformat $ "SIGHUP (" % shown % "): Cannot dynamically start/stop Kademlia."
-    changedMaxSubscrs = sformat $ "SIGHUP (" % shown % "): Cannot dynamically change maximum number of subscribers."
+    changedFormat     = sformat $ "SIGHUP ("%shown%"): Cannot dynamically change topology."
+    changedType       = sformat $ "SIGHUP ("%shown%"): Cannot dynamically change own node type."
+    changedKademlia   = sformat $ "SIGHUP ("%shown%"): Cannot dynamically start/stop Kademlia."
+    changedMaxSubscrs = sformat $ "SIGHUP ("%shown%"): Cannot dynamically change maximum number of subscribers."
 
     readFailed :: FilePath -> SomeException -> Text
     readFailed = sformat $ "SIGHUP: Failed to read " % shown % ": " % shown % ". Ignored."
 
     handlerError :: SomeException -> Text
-    handlerError = sformat $ "Exception thrown by staticPeersOnChange handler: " % shown % ". Ignored."
+    handlerError = sformat $
+        "Exception thrown by staticPeersOnChange handler: " % shown % ". Ignored."
 
-{-------------------------------------------------------------------------------
-  Interpreter
--------------------------------------------------------------------------------}
+----------------------------------------------------------------------------
+-- Interpreter
+----------------------------------------------------------------------------
 
 -- | Interpreter for the network config opts
-intNetworkConfigOpts :: forall m. (
-                         WithLogger     m
-                       , MonadIO        m
-                       , Mockable Fork  m
-                       , Mockable Catch m
-                       , Mockable Throw m
-                       )
-                     => NetworkConfigOpts
-                     -> m (T.NetworkConfig DHT.KademliaParams)
+intNetworkConfigOpts ::
+       forall m.
+       ( WithLogger m
+       , MonadIO m
+       , Mockable Fork m
+       , Mockable Catch m
+       , Mockable Throw m
+       )
+    => NetworkConfigOpts
+    -> m (T.NetworkConfig DHT.KademliaParams)
```

Comment:
This change is really useless, you should be more tolerant to formatting style you don't like.

Reply:
Yeah, maybe.